### ISSUE TITLE
Rename ssh methods

### ISSUE THREAD

**Issue by neur0manc** [gridscale/gridscale_api_client_python#12]
To make this client more in line with the [official API](https://gridscale.io/en/api-documentation/index.html#tag/SSH-key) I'd to propose a rename of the following methods:

- get_ssh_keys
- get_ssh_key
- create_ssh_key
- update_ssh_key
- delete_ssh_key
- get_ssh_key_events

`ssh_key` is canonically without the underscore in the API. The new function names would then be

- get_sshkeys
- get_sshkey
- create_sshkey
- update_sshkey
- delete_sshkey
- get_sshkey_events

For backwards-compat we could alias the methods to their old names.

It's a small annoyance, but I still wanted to mention it. I'm ready to work an a pull request if this change finds acceptance.

**Comment by bkircher** [comment]
I find the upper more correct (`get_ssh_keys` instead of `get_sshkeys`) since SSH key is really two words in English language and our API is just wrong here. But wouldn't be opposed to that as long as there are aliases.

**Comment by neur0manc** [comment]
Agreed, it’s more correct. Wanted to mention it anyway. It’s just unfortunate that it’s not named equally in the API and the API client.
